### PR TITLE
fix: change format of enabled linters

### PR DESCRIPTION
**Pull request by coolguy1771**
Signed-off-by: Tyler Witlin <tyler@codezero.io>

## Description

Changes the format of the enabled linters to enable all of them.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Technical improvement or removal of technical debt. 
- [x] Minor change: comments, documentation, trivial fixes

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

- [ ] Unit Tests

Optional:

- [ ] Functional Tests
- [ ] Integration Tests

### CODE REVIEW

**Review by Canuckaholic — CHANGES_REQUESTED** [review]
what's the reason for this format change?

**Comment by coolguy1771** [comment]
The way that megaliter parses arguments just requires it to be on one line, or else it doesn't enable any of the linters.

File: .github/workflows/megalinter.yaml [hunk 1]
```diff
@@ -24,15 +24,7 @@ jobs:
...
-          ENABLE_LINTERS: |-
-            ACTION_ACTIONLINT,
-            DOCKERFILE_HADOLINT,
-            GIT_GIT_DIFF,
-            JSON_JSONLINT,
-            MARKDOWN_MARKDOWNLINT,
-            MARKDOWN_MARKDOWN_TABLE_FORMATTER,
-            SPELL_MISSPELL,
-            CREDENTIALS_SECRETLINT
+          ENABLE_LINTERS: ACTION_ACTIONLINT, DOCKERFILE_HADOLINT, GIT_GIT_DIFF, JSON_JSONLINT, MARKDOWN_MARKDOWNLINT, MARKDOWN_MARKDOWN_TABLE_FORMATTER, SPELL_MISSPELL, CREDENTIALS_SECRETLINT
```
Comment (robblovell) [inline .github/workflows/megalinter.yaml:27]: I think it's more readable on separate lines...

**Review by robblovell — CHANGES_REQUESTED** [review]
I think we should leave this "as is". To me, it's more readable on multiple lines. :)

File: .github/workflows/megalinter.yaml (hunk 1 above)
Comment (robblovell) [reply .github/workflows/megalinter.yaml:27]: Oh, it's required to be on one line?

Comment (coolguy1771) [reply .github/workflows/megalinter.yaml:27]: Yes

Comment (robblovell) [reply .github/workflows/megalinter.yaml:27]: I see, it's a string. Have you tried `>` instead of `|-`?

Comment (coolguy1771) [reply .github/workflows/megalinter.yaml]: I have, it still only works with them all on one line

**Pull request closed without merging**